libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -239,10 +239,10 @@ AP_GPS::detect_instance(uint8_t instance)
 
 	// by default the sbf/trimble gps outputs no data on its port, until configured.
 	if (_type[instance] == GPS_TYPE_SBF) {
-		_broadcast_gps_type("SBF", instance, -1); // baud rate isn't valid
+		dstate->last_baud = -1; // baud rate isn't valid
```

Comment:
Same thing, this is unneeded, _broadcast_gps_type() didn't use to set it, and it doesn't matter for this GPS type, and can be entirely ignored.